README.md

```diff
@@ -43,44 +42,56 @@ How about:
 ![animated svg of the tldr client displaying the tar command](images/tldr.svg)
 
 This repository is just that: an ever-growing collection of examples
-for the most common UNIX, Linux, macOS, SunOS and Windows command-line tools.
+for the most common UNIX, Linux, macOS, SunOS, Android and Windows command-line tools.
 
 ## How do I use it?
 
 A popular and convenient way to access these pages on your computer
 is to install the [Node.js client](https://github.com/tldr-pages/tldr-node-client),
 which is supported by the tldr-pages project maintainers:
 
-    npm install -g tldr
+```sh
+npm install -g tldr
+```
+
+Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+```sh
+pip3 install tldr
+```
 
-    pip3 install tldr
+Then you have direct access to simplified, easy-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+If you just want to browse without an internet connection or installing software,
+check out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
 
-However, if you just want to browse without installing anything, check
-out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
+For browsing without installing a client to your computer,
+see the web client at <https://tldr.ostera.io>.
 
 There are also **various other clients** provided by the community,
 both for the command-line and for other platforms.
 For a comprehensive list of clients, head over to our [Wiki](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients).
 
-## How do I contribute?
+## How do I contribute to tldr-pages?
 
-- Your favourite command isn't covered?
-- You can think of more examples for an existing command?
+All contributions are welcome!
 
-All `tldr` pages are kept as Markdown files right here in this repository,
-so you can edit them directly and submit your changes as pull requests.
+Some ways to contribute include:
+
+- Adding your favorite command which isn't covered.
+- Adding examples or improving the content of an existing page.
+- Translating pages into different languages.
+
+All `tldr` pages are written in markdown, so they can be edited fairly simply and changes can be submitted in
+pull requests here using git on the command-line or 
+using the GitHub web interface
```

Comment:
I actually prefer my suggested wording to this one. Can you explain the reasoning g for the changes you made to it?

Reply:
I didn't like using Git on the command line wasn't included however the first part of the sentence I don't mind, either works for me.